app/pubspec.yaml

```diff
@@ -186,20 +186,17 @@ dev_dependencies:
   icons_launcher: ^3.0.0
   flutter_lints: ^5.0.0
   build_runner: ^2.4.12
-  convenient_test: ^1.3.7
-  convenient_test_dev: ^1.3.7
-  convenient_test_manager_dart:
-    git:
-      url: https://github.com/gnunicorn/flutter_convenient_test/
-      ref: ben-cli-limit-tests
-      path: packages/convenient_test_manager_dart
   msix: ^3.7.0
   uuid: ^4.0.0
-  enough_mail: ^2.1.6
+  # TODO: move to upstream version this once enough_mail is published
```

Comment:
I don't thinks this is used outside of the integration tests - this can be removed as well.